apps/google-analytics-4/README.md

```diff
@@ -1,27 +1,28 @@
-Contentful Marketplace App that displays page-based analytics from Google Analytics alongside Contentful content entries.
+# Google Analytics 4
+
+Contentful Marketplace App that displays real-time page-based analytics data from your organization’s Google Analytics 4 properties alongside relevant content entries.
 
 ## Requirements
 
-- [Google Analytics 4](https://developers.google.com/analytics/devguides/collection/ga4) (Note: Google's deprecated Universal Analytics is not supported with this verison of the Google Analytics App)
+- [Google Analytics 4](https://developers.google.com/analytics/devguides/collection/ga4) (Note: Google's deprecated Universal Analytics is not supported with this version of the Google Analytics App)
```

Comment:
"in the Google Analytics 4 app"

Just want to avoid the potential confusion of "version" when it's a holistically different thing.

Reply:
Ah it highlighted the whole like - that copy change was meant for after "supported!"